client/src/components/aggregations/CalendarWidget.js

```diff
@@ -63,6 +59,11 @@ const CalendarWidget = ({
       aspectRatio={3} // ratio of width-to-height
       ref={calendarComponentRef}
       events={events}
+      eventClick={info => {
+        history.pushState(info.event.url)
```

Comment:
Just `history.push(info.event.url)` please.

Reply:
Whoops, autocomplete not always a friend